src/caret_analyze/plot/bokeh/plot_factory.py

```diff
@@ -242,3 +243,39 @@ def _create_communication_period_plot_tuple(
         *communications: Communication
     ) -> CommunicationTimeSeriesPlot:
         return CommunicationPeriodPlot(communications)
+
+    @singledispatchmethod
+    def create_response_time_histogram_plot(arg) -> TimeSeriesPlot:
+        """
+        Get ResponseTimePlot instance.
+
+        Parameters
+        ----------
+        path : Collection[Path]
+            Target path.
+            This also accepts multiple path inputs by unpacking.
+
+        Returns
+        -------
+        ResponseTimePlot
+
+        """
+        raise InvalidArgumentError(f'Unknown argument type: {arg}')
+
+    @staticmethod
+    @create_response_time_histogram_plot.register
+    def _create_response_time_histogram_plot(
+        paths: Collection[Path],
+        case: str,
+        binsize_ns: int
+    ) -> ResponseTimePlot:
+        return ResponseTimePlot(paths, case, binsize_ns)
+
+    @staticmethod
+    @create_response_time_histogram_plot.register
+    def _create_response_time_histogram_plot_tuple(
+        *paths: Path,
+        case: str,
+        binsize_ns: int
```

Comment:
```suggestion
        *paths: Path,
        *,
        case: str = 'best-to-worst',
        binsize_ns: int = 10000000
```

Reply:
@hsgwa Could you please to tell me the meaning of the  * in 278 ?

Reply:
`*` means that `following arguments are keyword-only`
ref: https://docs.python.org/3/glossary.html#term-parameter

I intended as following.
```
create(a, b, c, ..., 'best', 100) # NG case: 'best' and 100 are included in paths tuple
create(a, b, c, ..., case='best', binsize_ns='100') # 'case' and 'binsize_ns' are recognized collectly.
```

Reply:
I see. Thank you for the answer.